lib/public_key/src/public_key.erl

```diff
@@ -226,6 +227,18 @@ pem_entry_encode('SubjectPublicKeyInfo',
     Spki = {'SubjectPublicKeyInfo',
             {'AlgorithmIdentifier', ?'id-dsa', ParamDer}, KeyDer},
     pem_entry_encode('SubjectPublicKeyInfo', Spki);
+pem_entry_encode('SubjectPublicKeyInfo',
```

Comment:
There are 2 function clauses that are exactly the same here could have been written as one clause like this:
```erlang
pem_entry_encode('SubjectPublicKeyInfo',
	 {#'ECPoint'{point = Key}, {namedCurve, ID}}) when is_binary(Key) andalso (ID == ?'id-Ed25519' orelse ?'id-Ed488' ) ->
    Spki = {'SubjectPublicKeyInfo',
	    {'AlgorithmIdentifier', ID, asn1_NOVALUE},
	    Key},
    pem_entry_encode('SubjectPublicKeyInfo', Spki);
```

Reply:
It could! In this case we avoid a slight code duplication by adding a guard instead of,  in my opinion, an easier to read direct match. We could add a function subject_public_info(ID) function also to avoid code duplication.

Reply:
I also removed the code duplication with help of a function when fixing the records usage discussed below.